src/components/countries/CountryLIst.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom'
+import { render, cleanup } from '@testing-library/react';
+import CountryList from './CountryList';
+
```

Comment:
alternatively, I saw you had a json file. you could import it and spread it here for a bit more accurate test than just 'f', either is fine.